cmd/utils/flags.go

```diff
@@ -1807,14 +1815,24 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		cfg.ShadowForkPeerIDs = ctx.GlobalStringSlice(ShadowforkPeersFlag.Name)
 		log.Info("Shadow fork peers", "ids", cfg.ShadowForkPeerIDs)
 	}
-	if ctx.GlobalIsSet(TxGossipBroadcastDisabledFlag.Name) {
-		cfg.TxGossipBroadcastDisabled = ctx.GlobalBool(TxGossipBroadcastDisabledFlag.Name)
-		log.Info("Transaction gossip broadcast disabled", "disabled", cfg.TxGossipBroadcastDisabled)
+	if ctx.GlobalIsSet(GossipTxBroadcastDisabledFlag.Name) {
+		cfg.GossipTxBroadcastDisabled = ctx.GlobalBool(GossipTxBroadcastDisabledFlag.Name)
+		log.Info("Gossip transaction broadcast disabled", "disabled", cfg.GossipTxBroadcastDisabled)
+	}
+	if ctx.GlobalIsSet(GossipTxReceivingDisabledFlag.Name) {
+		cfg.GossipTxReceivingDisabled = ctx.GlobalBool(GossipTxReceivingDisabledFlag.Name)
+		log.Info("Gossip transaction receiving disabled", "disabled", cfg.GossipTxReceivingDisabled)
+	}
+	if ctx.GlobalIsSet(GossipBroadcastToAllEnabledFlag.Name) {
+		cfg.GossipBroadcastToAllEnabled = ctx.GlobalBool(GossipBroadcastToAllEnabledFlag.Name)
+		log.Info("Gossip broadcast to all enabled", "enabled", cfg.GossipBroadcastToAllEnabled)
 	}
-	if ctx.GlobalIsSet(TxGossipReceivingDisabledFlag.Name) {
-		cfg.TxGossipReceivingDisabled = ctx.GlobalBool(TxGossipReceivingDisabledFlag.Name)
-		log.Info("Transaction gossip receiving disabled", "disabled", cfg.TxGossipReceivingDisabled)
+	// Only configure the gossip broadcast-to-all flag if --gossip.enablebroadcasttoall is set to true.
+	if ctx.GlobalIsSet(GossipBroadcastToAllCapFlag.Name) && cfg.GossipBroadcastToAllEnabled {
```

Comment:
I think this can be set together with the `GossipTxReceivingDisabledFlag` as both together are used as configuration anyway. So by adding the default value for `GossipBroadcastToAllCap` we can just print it above as well